src/app/clusters/ota-provider/ota-provider-delegate.h

```diff
@@ -31,9 +31,9 @@ namespace clusters {
 class OTAProviderDelegate
 {
 public:
-    // TODO: protocolsSupported should be list of OTADownloadProtocol enums, not uint8_t*
+    // TODO(#8605): protocolsSupported should be list of OTADownloadProtocol enums, not uint8_t
     virtual EmberAfStatus HandleQueryImage(uint16_t vendorId, uint16_t productId, uint16_t imageType, uint16_t hardwareVersion,
-                                           uint32_t currentVersion, uint8_t * protocolsSupported, const chip::ByteSpan & location,
+                                           uint32_t currentVersion, uint8_t protocolsSupported, const chip::ByteSpan & location,
```

Comment:
Shouldn't this be the actual enum type?  Or do we codegen this with uint8_t instead of the right type?

Reply:
It's codegen as `uint8_t` (you can look at `callback.h`)

Reply:
This should be addressed in #8313